src/js/npm/redis/RedisClient.hx

```diff
@@ -138,6 +147,20 @@ extern class RedisClientBase<TSelf:RedisClientBase<TSelf,TReturn>, TReturn> exte
   @:overload(function(password:RedisString):TReturn {})
   function auth(password:RedisString, callback:VoidCallback):TReturn;
 
+  /**
+   * Duplicate all current options and return a new redisClient instance.
+   * All options passed to the duplicate function are going to replace the
+   * original option. If you pass a callback, duplicate is going to wait
+   * until the client is ready and returns it in the callback. If an error
+   * occurs in the meanwhile, that is going to return an error instead in
+   * the callback.
+   * @return RedisClient
+   */
+  @:overload(function():RedisClient {})
+  @:overload(function(callback:Null<js.Error>->RedisClient->Void):RedisClient {})
+  @:overload(function(opts:js.npm.Redis.RedisOptions):RedisClient {})
+  function duplicate(opts:js.npm.Redis.RedisOptions, callback:Null<js.Error>->RedisClient->Void):RedisClient;
```

Comment:
Could you take off this one? It's already in our API, so merging it will cause a duplicate function error